doc/classes/Transform2D.xml

```diff
@@ -172,10 +172,13 @@
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Transform2D( 1, 0, 0, 1, 0, 0 )">
+			[Transform2D] with no translation, rotation or scaling applied. When applied to other data structures, [constant IDENTITY] performs no transformation.
 		</constant>
 		<constant name="FLIP_X" value="Transform2D( -1, 0, 0, 1, 0, 0 )">
+			[Transform2D] with vertical mirroring applied.
```

Comment:
I suppose this is correct (I haven't played with transforms much), but are you sure it's vertical mirroring here?

Reply:
I was pretty sure until you mentioned it, let me run a quick test locally.

Reply:
Ok `FLIP_X` mirrors left-right, `FLIP_Y` mirrors up-down.

I think the language is ambiguous. I'll change it up.

Reply:
I think it's more clear, albeit more technical now.